book.cmd/src/main/java/com/happystays/book/cmd/infrastructure/BookCommandEventStore.java

```diff
@@ -27,32 +30,43 @@ public class BookCommandEventStore implements EventStore {
 
     @Override
     public void saveEvents(String aggregateId, BaseEvent baseEvent, int expectedVersion) {
```

Comment:
Currently since we are passing BaseEvent, Can't make this iterable for future scope and do iteration while creating the object